src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs

```diff
@@ -683,6 +697,14 @@ private void SetBuildpacksFromManifest(AppManifest appManifest)
                 }
             }
         }
+        private void SetStartCommandFromManifest(AppManifest appManifest)
+        {
+            var startCmmd = appManifest.Applications[0].Command;
+            if (!string.IsNullOrWhiteSpace(startCmmd))
+            {
+                StartCommand = startCmmd;
+            }
+        }
     }
```

Comment:
I'm thinking we might want to set StartCommand to null if it's either null or whitespace to account for:
1. if a manifest is provided with no start command (clear any previous value to sync with the new manifest)
2. if a manifest is provided with whitespace for a start command (don't keep the whitespace in the new manifest)... (is this even a real scenario though?)

```suggestion
        private void SetStartCommandFromManifest(AppManifest appManifest)
        {
            var startCmmd = appManifest.Applications[0].Command;
           
            StartCommand = string.IsNullOrWhiteSpace(startCmmd) ? null : startCmmd;
        }
    }
```